### PR TITLE
remove deprecated mentions of signrawtransaction from fundraw help

### PR DESCRIPTION
**Pull request by instagibbs**
RPC call has been removed as of 0.17.99.

### CODE REVIEW

**Comment by instagibbs** [comment]
Removed all mentions of signrawtransaction I could find(that weren't documentation says it was being removed or not to make RPC calls like it)

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -3042,9 +3043,9 @@ static UniValue fundrawtransaction(const JSONRPCRequest& request)
                             "\nAdd sufficient unsigned inputs to meet the output value\n"
                             + HelpExampleCli("fundrawtransaction", "\"rawtransactionhex\"") +
                             "\nSign the transaction\n"
-                            + HelpExampleCli("signrawtransaction", "\"fundedtransactionhex\"") +
+                            + HelpExampleCli("signrawtransactionwithwallet", "\"fundedtransactionhex\"") +
                             "\nSend the transaction\n"
-                            + HelpExampleCli("sendrawtransaction", "\"signedtransactionhex\"")
+                            + HelpExampleCli("sendrawtransactionwithwallet", "\"signedtransactionhex\"")
```
Comment (MarcoFalke) [inline src/wallet/rpcwallet.cpp]: Needs rebase

**Comment by instagibbs** [comment]
rebased

**Comment by fanquake** [comment]
@instagibbs Looks like you've lost the other `signrawtransaction` changes.

**Comment by instagibbs** [comment]
@fanquake fixed, and noticed I was typing "withprivkey" sometimes, so fixed those as well.

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -3077,9 +3078,9 @@ static UniValue fundrawtransaction(const JSONRPCRequest& request)
...
```
Comment (kristapsk) [inline src/wallet/rpcwallet.cpp]: It's still `sendrawtransaction`, not `sendrawtransactionwithwallet`.

**Review by kristapsk — CHANGES_REQUESTED** [review]

**Review by kristapsk — APPROVED** [review]
utACK 851380ce177bccf0d608fa8017a383f2ac273dc1

**Comment by fanquake** [comment]
utACK 851380c

**Pull request merged**